src/parser/plugins/typescript.ts

```diff
@@ -367,6 +367,7 @@ function tsParseMappedType(): void {
 function tsParseTupleType(): void {
   expect(tt.bracketL);
   while (!eat(tt.bracketR) && !state.error) {
+    // Do not validate presence of either none or only labeled elements
```

Comment:
This comment doesn't seem super-necessary; across the parser code, all validation is skipped.

Reply:
Put it here because it replaces a large chunk of code in babel, but agree that it's pretty useless outside the context of porting babel changes